src/utils/crypto.ts

```diff
@@ -0,0 +1,148 @@
+import type { IEncryptionResult } from '@/types';
+import { STORAGE_KEYS, PASSWORD_ENCRYPTION_ALGO, IS_EXTENSION } from '@/constants';
+import { WalletStorage } from '@/lib/WalletStorage';
```

Comment:
Wallet storage should not be used within util function as this breaks functional programming principles.

Reply:
Fixed.